src/redux/news/news.actions.js

```diff
@@ -0,0 +1,12 @@
+import { NEWS_LOADED, SET_NEWS } from './news.types';
+
+const setNews = (news) => ({
+  type: SET_NEWS,
+  payload: news
+});
+
+const watchNewsLoad = () => ({
```

Comment:
Name of type should match with the name of action creator

Reply:
renamed to GET_NEWS and getNews